woocommerce/wc-admin
# Fix Table component `onQueryChange` prop default

Fixes #4916.

This PR updates the default value for the `onQueryChange` prop of the `Table` component to return a function that returns a noop, rather than using Lodash's `noop()`.

### Detailed test instructions:

- Either add a `<Table />` component to a plugin or existing view that doesn't define the `onQueryChange` prop
- or
- Comment out these lines: https://github.com/woocommerce/woocommerce-admin/blob/9cbfe53a2ef799705a44d183bd8c07cb5fcf2ef0/packages/components/src/table/docs/example.js#L60-L65 and visit the Table DevDocs
- Change the table page or number of results per page
- Verify no console errors



### Changelog Note:



Fix: JavaScript error with Table component default prop value.
